src/benchmarks/real-world/Roslyn/CompilerBenchmarks.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsShipping>false</IsShipping>
+    <StrongNameKeyId>MicrosoftShared</StrongNameKeyId>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
I do not see these two files in the PR. Is this intended?

Reply:
Ah, I deleted them because they were obsolete and forgot to remove them from the project file. Thanks.